service/src/main/kotlin/fi/nokiankaupunki/evaka/invoice/service/ProEInvoiceGenerator.kt

```diff
@@ -262,6 +264,7 @@ class ProEInvoiceGenerator(private val invoiceChecker: InvoiceChecker, val finan
             it.value.forEach {
                 result += generateRow(rowHeaderRowFields, it)
                 result += generateRow(detailRowFields, it)
+                result += generateRow(descriptionRowFields, it)
```

Comment:
I think the row should be added only when `description.isNotBlank()`

Reply:
Excellent point, description row is now only shown when not blank